cmd/fuzzbinder/main.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main contains a command-line tool for generating fuzzing claims
+// for a revision of the source code.
+package main
+
+func main() {
+}
```

Comment:
I suggest adding something like `fmt.Println("Not yet implemented!")` to be clear that some functionality will be added here.

Reply:
Using `fmt.Println` causes a linting error. It's included in forbidigo.

Reply:
Sorry, you could use `log.Println` or `go.Printf`.